packages/react-sdk/src/components/BoardBar/pdf/utils.ts

```diff
@@ -40,7 +40,68 @@ export function image(element: ImageElement, base64content: string): Content {
   };
 }
 
+/**
+ * This ensures that the svg is meeting the following conditions:
+ *
+ * - The width and height are set (required for the canvas to work)
+ * - The svg is actually an svg
+ *   - If the svg is not an svg we return the original string
+ *   - If the svg is detected to be a png or jpeg we return the original string but set the mimetype accordingly
+ *
+ * @param element The element that is being processed
+ * @param svg The base64 encoded svg string
+ * @returns The base64 encoded svg string after the preprocessing
+ */
+function preprocessSvg(element: ImageElement, svg: string): string {
+  if (element.mimeType !== 'image/svg+xml') {
+    return svg;
+  }
+
+  // Convert the svg base64 to the actual svg
+  const svgString = atob(svg);
+
+  // We lied in the past about the mimetype, so we need to check if the svg is actually an svg and otherwise return the original string
+  if (!svgString.includes('<svg')) {
+    element.mimeType = svgString.includes('PNG') ? 'image/png' : 'image/jpeg';
+    return svg;
+  }
+
+  // Parse the svg string
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(svgString, 'image/svg+xml');
+  const svgElement = doc.documentElement;
+
+  // Check if a size is set and if not set it to the size from the viewbox
+  if (!svgElement.getAttribute('width') || !svgElement.getAttribute('height')) {
+    const viewBox = svgElement.getAttribute('viewBox')?.split(' ');
+    if (viewBox) {
+      svgElement.setAttribute('width', viewBox[2]);
+      svgElement.setAttribute('height', viewBox[3]);
+    } else {
+      svgElement.setAttribute('width', element.width.toString());
+      svgElement.setAttribute('height', element.height.toString());
+    }
+  }
+
+  // Convert the svg element to a string
+  const serializer = new XMLSerializer();
+  const svgStringProcessed = serializer.serializeToString(svgElement);
+
+  // Convert the svg string to base64
+  return btoa(svgStringProcessed);
+}
+
 export function conv2png(element: ImageElement, base64content: string): string {
+  // If we have a svg image we need to preprocess it for the canvas to work
+  if (element.mimeType === 'image/svg+xml') {
```

Comment:
The mime type check is now doubled. Because `preprocessSvg` also explains it in it's description, maybe drop it here? No strong opinion…